frontend/app/abtests/PayPalTestVariants.scala

```diff
@@ -0,0 +1,10 @@
+package abtests
+
+import scala.util.Random
+
+object PayPalTestVariants {
+  val withPayPal = "with_paypal"
+  val stripeOnly = "stripe_only"
+  val variants = List(withPayPal, stripeOnly)
```

Comment:
It can be good to avoid stringly-typed stuff like this with an enum-style class, like [`BundleVariant`](https://github.com/guardian/membership-frontend/blob/37008909/frontend/app/abtests/BundleVariant.scala#L5-L26).

In this case it's only a bit of nicety tho', as unlike `BundleVariant`, there isn't a lot of extra pricing and behaviour associated, so not essential for this PR. Might be a nice excuse to try out [lloydmeta/enumeratum](https://github.com/lloydmeta/enumeratum) tho'!